### ISSUE TITLE
In installer.py one of the raise statements is unreachable

### ISSUE THREAD

**Issue by dylanmtaylor** [archlinux/archinstall#465]
Line 449:
```
raise RequirementError(f"Could not identify the UUID of {self.partition}, there for {self.target}/boot/loader/entries/arch.conf will be broken until fixed.")
```

This is unreachable; all the paths return before making it to this statement. I think this should be removed or rewritten so it's actually working as intended. Not sure what the ideal solution is.

**Comment by Torxed** [comment]
Correct, if it's this line we're talking about:
https://github.com/archlinux/archinstall/blob/d0bc166e7e9f381be15791ebd1d0150ba17a66c0/archinstall/lib/installer.py#L449

Then it's 100% unreachable and belongs to legacy code when we had issues with catching partitions reliably.

**Comment by dylanmtaylor** [comment]
Fixed in #471
